packages/core/lib/commands/debug/inlineConfigNetwork.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function (options) {
+  const Config = require("@truffle/config");
+
+  const configObject = Config.default();
+  configObject.networks = {
+    inline_config: {
+      url: options.url,
+      network_id: "*"
+    }
+  };
+  configObject.network = options.network ? options.network : "inline_config";
```

Comment:
Why are we doing this? In this flow, network will never be set before reaching this line. Will this not be the case some time in the future?

Reply:
fair point. I believe we can remove that